src/main/java/uk/gov/companieshouse/itemgroupworkflowapi/model/ItemDescriptionIdentifier.java

```diff
@@ -17,7 +17,7 @@ public enum ItemDescriptionIdentifier {
             .collect(Collectors.toMap(ItemDescriptionIdentifier::toString, Function.identity()));
     }
 
-    private ItemDescriptionIdentifier(String identifierName) {
+    ItemDescriptionIdentifier(String identifierName) {
```

Comment:
Please define access identifier, make it protected

Reply:
Every enum constructor without an access modifier is private. 
Using the public or protected modifiers will cause a compilation error. It's only possible to define private constructors on an enum type without causing compilation errors.